src/styles/tailwind.scss

```diff
@@ -579,6 +579,8 @@ video {
 
```

Comment:
Since this is a generated file, do we want it in the code base? Should it be generated either when we deploy it to NPM or when the user builds it in their own project.